client/src/Components/InterviewHeader.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { useStyles } from '../themes/theme';
+import Grid from '@material-ui/core/Grid';
+import LanguageSelectMenu from './LanguageSelectMenu'
+import { ExitInterviewButton } from '../components/Buttons'
+import { useHistory } from 'react-router-dom'
+
+const InterviewHeader = (props) => {
+  const classes = useStyles();
+  const history = useHistory()
```

Comment:
I noticed that you are inconsistent with the semi-colons, which probably means that pretttier/eslint is not set up